recipe/meta.yaml

```diff
@@ -6,36 +6,59 @@ package:
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/gravis-{{ version }}.tar.gz
-  sha256: a36342602aa7da3bbd674c970439717926a8f7ca0678aa82cc5d756e8c2a529b
+  # switch to GH release archive, because of tests
+  url: https://github.com/robert-haas/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 8734066afd749fd85f9f173f92a1b501ae8e31723d1894c15f29fcec3eac5419
+  patches:
+    - patches/0001_fix-win-unicode-encoding.patch  # [win]
 
 build:
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
   number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
+  build:
+    - m2-patch    # [win]
   host:
-    - python >=3.5
+    - python
+    - setuptools
     - pip
+    - wheel
   run:
-    - python >=3.5
+    - python
+    # access to package files
     - setuptools >=40.0
```

Comment:
`setuptools` is not a run dependency

Reply:
Yeah, it's not needed.

Reply:
Here is a additional info about dependencies https://robert-haas.github.io/gravis-docs/rst/installation.html#installation

Reply:
ah, I see. Ok I will remove it.

Note: it's in `setup.py` but never used besides it:
https://github.com/robert-haas/gravis/blob/3e63b606141bf78617b1247048a9e3ccca6d5824/setup.py#L128

Reply:
```suggestion
```